.github/workflows/unit.yml

```diff
@@ -79,7 +79,7 @@ jobs:
         sudo rm /etc/apt/sources.list.d/* && sudo apt-get update
         sudo apt-get --no-install-recommends install -y ninja-build lcov
     - name: Build Simulation
-      run: ./script/cmake-build simulation
+      run: ./script/cmake-build simulation -DOT_MULTIPAN_RCP=ON
```

Comment:
I think that is preferable, but there didn't seem to be a reference for including configurable changes in the test suite. It makes sense to have both cases, and it seems to be a relatively low impact on the workflow, but it was not clear to me if this would be endorsed by the community.

I will update the workflow, and evaluate the checks to see if the change is as I expect.